sources/packages/forms/src/form-definitions/sfaa2025-26.json

```diff
@@ -18442,12 +18595,12 @@
       "key": "page6",
       "tags": [],
       "properties": {},
-      "customConditional": "",
+      "customConditional": "show = data.dependantstatus === \"dependant\";",
```

Comment:
Same thing. simple condition can be used. 
![image](https://github.com/user-attachments/assets/f4fdced0-85c0-4a81-b54f-bbeec2a96ad2)